cabal.project

```diff
@@ -95,8 +95,8 @@ source-repository-package
 source-repository-package
     type: git
     location: https://github.com/input-output-hk/cardano-addresses
-    tag: 46e3807ad4529344567c50ee5c3489e60575a9dd
-    --sha256: 03ya5md876sfw0nihmjk5rza76z1m9j7q34j8rj2marrkb51vhf0
+    tag: 6b55f96d57a181f898eb2a50531d3ae4280c549c
```

Comment:
I believe our convention is to use the latest _released_ version of cardano-addresses, so perhaps we should use https://github.com/input-output-hk/cardano-addresses/commit/b4a9765314aee3f2926a8eca79f6cfdac836bda1 instead. Not sure if this is that important though.